weather_forecast_retrieval/data/hrrr/file_loader.py

```diff
@@ -100,16 +98,11 @@ def get_saved_data(self,
 
         self.force_zone_number = force_zone_number
 
-        if output_dir is not None:
-            self.output_dir = output_dir
+        if input_dir is not None:
+            self.output_dir = input_dir
```

Comment:
So why change to `input_dir` to set it as `output_dir`? Could this just get renamed to something more descriptive? This path is the directory to the HRRR data correct, so something like `hrrr_dir` or `hrrr_file_directory`?

Reply:
Good catch. I should rename this property as well.
From my perspective, `output_dir` is misleading and looks like a copy&paste reminiscence.

Reply:
I would like to keep the property name a little more neutral. How about `file_dir` as we are loading with the `FileLoader` class?